ocs_ci/ocs/resources/pod.py

```diff
@@ -1176,3 +1176,20 @@ def get_noobaa_pods(noobaa_label=constants.NOOBAA_APP_LABEL, namespace=None):
     noobaa_pods = [Pod(**noobaa) for noobaa in noobaas]
 
     return noobaa_pods
+
+
+def wait_for_dc_app_pods_to_reach_running_state(dc_pod_obj):
+    """
+    Wait for DC app pods to reach running state
+
+    Args:
+        dc_pod_obj (list): list of dc app pod objects
+    """
+    for pod_obj in dc_pod_obj:
+        name = pod_obj.name[:-8]
```

Comment:
you can get pod name from pod_obj.get_labels().get('name')

Reply:
Done